FinSearchUnified/BusinessLogic/Models/FindologicArticleModel.php

```diff
@@ -470,13 +470,18 @@ protected function setImages()
 
     protected function setSales()
     {
-        $orderDetailQuery = $this->orderDetailRepository->createQueryBuilder('order_details')
-            ->groupBy('order_details.articleId')
-            ->where('order_details.articleId = :articleId')
-            ->select('sum(order_details.quantity)')
-            ->setParameter('articleId', $this->baseArticle->getId());
+        $pseudoSales = $this->baseArticle->getPseudoSales();
+        if (is_int($pseudoSales) && $pseudoSales > 0) {
```

Comment:
As far as the docblock suggests, this may never return anything else than `int`:

![image](https://user-images.githubusercontent.com/20555112/99394216-cfdab600-28de-11eb-8539-ace7ca6b48eb.png)

In case you want to check for nullable, please be explicit by checking for `$pseudoSales !== null`.